src/clients/http_client/__tests__/http_client.test.ts

```diff
@@ -828,16 +854,16 @@ describe('HTTP client', () => {
         body: {errors: 'Error 500'},
         code: 500,
         statusText: 'Error 500',
-        headers: {'X-Text-Header': ['Error 500']},
+        headers: {'X-Text-Header': 'Error 500'},
```

Comment:
(1/3) I'm a bit concerned about removing the `[]` from the comparison to get this to pass ... but this is the format we're expecting/intending to be returned for headers?

Reply:
Now using `canonicalizeHeaders()` when queuing a mocked response.